firebaseFunctions/src/helpers.ts

```diff
@@ -159,6 +161,92 @@ export const updateDealUpdatesCollection = (dealId: string): void => {
   });
 };
 
+const prepareAvatarUrl = (url: string): string => {
```

Comment:
ar1. The API returns a mix of full URL (https://.../avatar.png) and only file name (avatar.png). This function unifies the URLs to always have the full URL. I've changed the function name to `unifyAvatarUrl` in the new PR: #1173.
ar2. We can't ensure the format won't change. I guess we need to trust DeepDAO for keeping their storage location constant. But in case it changes- I've added a constant for simple adjustment.
ar3. Unfortunately, there's no documentation for URLs (or anything).